### ISSUE TITLE
reproducing results using pretrained Deep Fashion Model -- quality seems not as good as expected

### ISSUE THREAD

**Issue by nikjetchev** [Ha0Tang/XingGAN#10]
Dear paper authors

I am working on a NeurIPS paper for PoseMorphing, so I wanted a good comparison with your state-of-the-art method. 

I tried to run your pretrained deepfashion model, and it worked. However, the results seem worse than I expected, can it be that the latest pytorch 1.7 has broken some detail in your version?

I write the command I used to run your model, and 2 example results. Maybe you can tell me if they look as expected, or something has gone wrong?

```
python test.py --dataroot deepfashion/ --name deepfashion_XingGAN --model XingGAN --phase test --dataset_mode keypoint --norm instance --batchSize 1 --resize_or_crop no --gpu_ids 0 --BP_input_nc 18 --no_flip --which_model_netG Xing --checkpoints_dir ./checkpoints --pairLst /deepfahion/fasion-resize-pairs-test.csv --which_epoch latest --results_dir ./results --display_id 0
```

fashionMENTees_Tanksid0000730104_7additional jpg___fashionMENTees_Tanksid0000730104_1front jpg_vis
fashionWOMENBlouses_Shirtsid0000337203_3back jpg___fashionWOMENBlouses_Shirtsid0000337203_2side jpg_vis

![fashionMENTees_Tanksid0000730104_7additional jpg___fashionMENTees_Tanksid0000730104_1front jpg_vis](https://user-images.githubusercontent.com/10964241/115688541-8bef4f80-a35b-11eb-87f8-6652d3b33252.jpg)

![fashionWOMENBlouses_Shirtsid0000337203_3back jpg___fashionWOMENBlouses_Shirtsid0000337203_2side jpg_vis](https://user-images.githubusercontent.com/10964241/115688590-94478a80-a35b-11eb-83bd-2dd3440fc3e3.jpg)

thanks a lot for your help to make research reproducible

**Comment by Ha0Tang** [comment]
The results are worse than what I expected, it may be caused by a different environment.

In addition, you can also try [BiGraphGAN](https://github.com/Ha0Tang/BiGraphGAN).

**Comment by nikjetchev** [comment]
Thanks for clarifying that.
It is not so easy to reconstruct exactly the code environment you had when running XingGAN and getting good results.
But do you have an archive folder of test results, the 8570 generate images supposed to be in the DeepFashion test set?
If you have this data, I can still run a clean comparison and cite your paper.

thanks a lot for the help

**Comment by Ha0Tang** [comment]
This project was done last year, and the generated results have not been preserved. You can absolutely use the results you get to compare in your paper. 

Good luck with your NeurIPS paper!
